app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -315,6 +315,7 @@ class StateFragmentPresenter @Inject constructor(
 
     viewModel.itemList.clear()
     viewModel.itemList += pendingItemList
+    binding.stateRecyclerView.smoothScrollToPosition(0)
```

Comment:
Check PR #408 for this and merge that in this PR.

Reply:
Done